benchmark/script.py

```diff
@@ -95,14 +85,14 @@ class BenchmarkOptions(Options):
         nodename=node(),
         platform=platform(),
     ),
-    scenario=config['scenario'],
-    operation=config['operation'],
-    metric=config['metric'],
+    scenario=scenario_config.copy(),
+    operation=operation_config.copy(),
+    metric=metric_config.copy(),
 )
 
```

Comment:
Could you add a comment to explain why we choose to copy the dictionaries at this point? Either that or provide some documentation for `driver` that explains that the different `*_config` parameters will be modified.